mongodbatlas/clusters.go

```diff
@@ -408,6 +420,51 @@ func (s *ClustersServiceOp) GetProcessArgs(ctx context.Context, groupID, cluster
 	return root, resp, err
 }
 
+// LoadSampleDataset loads the sample dataset into your cluster.
+// See more: https://docs.atlas.mongodb.com/reference/api/cluster/load-dataset/
```

Comment:
[super-nit] if you don't add the empty line in renders as a single line
```suggestion
// LoadSampleDataset loads the sample dataset into your cluster.
//
// See more: https://docs.atlas.mongodb.com/reference/api/cluster/load-dataset/
```